cmd/facts.go

```diff
@@ -81,17 +81,31 @@ func gather(*cobra.Command, []string) {
 	var argument = viper.GetString("argument")
 	var pluginsFolder = viper.GetString("plugins-folder")
 
-	engine := factsengine.NewFactsEngine("", "")
+	gathererManager := gatherers.NewManager(map[string]gatherers.FactGatherer{
+		gatherers.CorosyncFactKey: gatherers.NewDefaultCorosyncConfGatherer(),
```

Comment:
Could we maybe have a function that returns all the known gatherers from the `gatherer` package? That returns a map. Otherwise, I'm afraid we will forget of adding the new gatherers here (and in the next function)

Reply:
done, it's called StandardGatherers